src/Makefile

```diff
@@ -0,0 +1,16 @@
+CFLAGS=	-fPIC
```

Comment:
Please add these files to the CMake build system (CMakeLists.txt) rather than a Makefile.